### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by calebnelton**
Had some spare time ...

### CODE REVIEW

File: src/Microsoft.AspNetCore.Identity/UserManager.cs [hunk 1]
```diff
@@ -1951,7 +1951,7 @@ public virtual Task<IList<TUser>> GetUsersInRoleAsync(string roleName)
             var store = GetUserRoleStore();
             if (roleName == null)
             {
-                throw new ArgumentNullException("role");
+                throw new ArgumentNullException(nameof(roleName));
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Identity/UserManager.cs]: Nice.

**Comment by Eilon** [comment]
@calebnelton can you squash into one commit?

@HaoK can you review?

**Comment by calebnelton** [comment]
@Eilon sorry, had to learn how to squash. Hopefully I did it right.

**Comment by Eilon** [comment]
@calebnelton looks pretty squashed to me :smile:

**Comment by MaximRouiller** [comment]
Just like a pug. 

I'll see myself out.

**Comment by Eilon** [comment]
@MaximRouiller I'd like two tickets to your next performance, kthxbye!

**Comment by MaximRouiller** [comment]
Be there at the 2016 Summit and we'll :beers: to that disgusting pun I did with the pug. :smile:

**Comment by HaoK** [comment]
Merged d1585617f9147daa2d8a29c2efb04f0eeb49d5ae

Thanks!

**Pull request closed without merging**